### PR TITLE
updates to titanium

### PR DESCRIPTION
**Pull request by ffMathy**
Losened some types that were too strong.

### CODE REVIEW

**Comment by ffMathy** [comment]
This is not "unowned". It clearly has listed owners (me included). How do we fix that?

File: types/alloy/index.d.ts [hunk 1]
```diff
@@ -148,7 +145,7 @@ interface AlloyInterface {
    * @param name Name of controller to instantiate.
    * @param args Arguments to pass to the controller
    */
-  createController(name: string, args?: any): AlloyController;
+  createController(name: string, args?: any): AlloyController & any;
```
Comment (sheetalkamat) [inline types/alloy/index.d.ts]: `AlloyController & any` = `any` .. Either use specific type or `any` if its suppose to be anything.

Comment (sheetalkamat) [reply types/alloy/index.d.ts]: Refer to https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/alloy/index.d.ts (hunk 1 above)
Comment (ffMathy) [reply types/alloy/index.d.ts]: Fixed! Sorry about that. That was a brainfart.

**Comment by janvennemann** [comment]
@ffMathy please make sure to reflect the changes to the Titanium typings you made here back into our definition generator. Otherwise they will be overwritten the next time we auto generate them from our docs.

**Comment by ffMathy** [comment]
@ffMathy can you link to that definition generator?

**Comment by janvennemann** [comment]
Sure, you can find it here: [typescript_generator.js](https://github.com/appcelerator/docs-devkit/blob/develop/packages/titanium-docgen/generators/typescript_generator.js)

**Pull request merged**